clustered/server/src/main/java/org/ehcache/clustered/server/store/ClusterTierActiveEntity.java

```diff
@@ -133,6 +139,8 @@
   private volatile List<InvalidationTuple> inflightInvalidations;
   private final Set<ClientDescriptor> connectedClients = ConcurrentHashMap.newKeySet();
   private final int chainCompactionLimit;
+  private volatile Long dataSizeThreshold = null;
```

Comment:
This could be a primitive type initialized right away. `private final long dataSizeThreshold = Long.getLong(SYNC_DATA_SIZE_PROP, DEFAULT_SYNC_DATA_SIZE_THRESHOLD);`

Reply:
The current code reads it late. So I wanted to keep it that way (in case we allow system propeties to be set runtime through JMX etc).. I fixed a bug in that area so that I re-read the system property on start of every sync (i.e concurrency key 0).

Reply:
I think it would be super duper overkill if that was the original intent.